tasks/getSpotifyToken.js

```diff
@@ -0,0 +1,38 @@
+const { Task } = require('klasa');
+const fetch = require('node-fetch');
+
+// Login to https://developer.spotify.com/dashboard/applications
+// Create an app & get your clientID/secret, place them below
+// Run every 1 hour to refresh: this.client.schedule.create("getSpotifyToken", "*/60 * * * *")
+const clientID = '';
+const clientSecret = '';
+
+const authorization = Buffer.from(`${clientID}:${clientSecret}`).toString('base64');
+
+module.exports = class extends Task {
+
+	async run() {
+		this._getToken();
+	}
+
+	async init() {
+		this._getToken();
+	}
+
+	async _getToken() {
+		const accessToken = await fetch(`https://accounts.spotify.com/api/token`, {
+			method: 'POST',
+			body: 'grant_type=client_credentials',
+			headers: {
+				Authorization: `Basic ${authorization}`,
+				'Content-Type': 'application/x-www-form-urlencoded'
+			}
+		})
+			.then(response => response.json())
+			.then(response => response.access_token)
+			.catch(console.error);
```

Comment:
This should use a try/catch. And use `this.client.emit('wtf', error);` instead. The catch logs in an inconsistent style (no timestamp, no shard ID...) and makes accessToken be assigned to undefined, it should be assigned to null.